tox.ini

```diff
@@ -26,3 +33,30 @@ deps =
     pylint
 whitelist_externals =
     sh
+
+[testenv:setup]
+basepython = python3
+envdir={toxworkdir}/acceptance
+setenv =
+  CONFIG_PATH = ~/.wazo-acceptance/config.yml
+commands =
+  bash -c 'if [ ! -e {env:CONFIG_PATH} ]; then \
+    mkdir -p $(dirname {env:CONFIG_PATH}); \
+    echo -e "default:\n  wazo_host: {posargs}\n" > {env:CONFIG_PATH}; \
+  fi'
+  behave features/pre_daily --verbose
+  wazo-acceptance -v -p
+whitelist_externals =
+  bash
+
+[testenv:acceptance]
```

Comment:
What is the purpose of this environment? I was thinking that the goal of tox was to automate the process, not to wrap the cli with the same amount of flexibility, Am I wrong?

Reply:
Since I known now that wazo-acceptance is useful only for the setup I can remove this target.